staticDocs/Permissions.md

```diff
@@ -0,0 +1,151 @@
+| updateFormElementList()  | Form Element Lists | Manager          |
+
+### Form Element Lookups
+
+| Endpoint                   | Permission           | Permission Level   |
+| -------------------------- | -------------------- | ------------------ |
+| createFormElementLookup()  | Form Element Lookups | Manager            |
+| deleteFormElementLookup()  | Form Element Lookups | Manager            |
+| getFormElementLookup()     | Form Element Lookups | Manager, Read Only |
+| searchFormElementLookups() | ---                  | ---                |
+| updateFormElementLookup()  | Form Element Lookups | Manager            |
+
+### Forms
+
+| Endpoint                          | Permission       | Permission Level   |
+| --------------------------------- | ---------------- | ------------------ |
+| createForm()                      | Forms            | Manager            |
+| createSubmissionAttachment()      | FaaS             | Developer          |
+| deleteForm()                      | Forms            | Manager            |
+| executeWorkflowEvent()            | Form Submissions | Manager            |
+| generateFormUrl()                 | Forms\*          | Manager, Read Only |
+|                                   | Apps             | Manager, Read Only |
+|                                   | FaaS\*\*         | Developer          |
+| generateSubmissionAttachmentUrl() | Form Submissions | Manager, Read Only |
+| generateSubmissionDataUrl()       | Form Submissions | Manager, Read Only |
+| generateWorkflowAttachmentLink()  | Form Submissions | Manager, Read Only |
+| getForm()                         | Form             | Manager, Read Only |
+| getFormSubmissionMeta()           | Form Submissions | Manager, Read Only |
+| getSubmissionAttachmentBuffer()   | Form Submissions | Manager, Read Only |
+| getSubmissionAttachmentMeta()     | Form Submissions | Manager, Read Only |
+| getSubmissionAttachmentStream()   | Form Submissions | Manager, Read Only |
+| getSubmissionData()               | Form Submissions | Manager, Read Only |
+| migrateForm()                     | Form             | Manager            |
+| searchForms()                     | Form             | Manager, Read Only |
+| searchSubmissions()               | Form Submissions | Manager, Read Only |
+| updateForm()                      | Form             | Manager            |
+| uploadEmailAttachment()           | FaaS             | Developer          |
+| decryptUserToken()                | ---              | ---                |
+| encryptUserToken()                | ---              | ---                |
+| generateFormElement()             | ---              | ---                |
+| generatePageElement()             | ---              | ---                |
+| validateForm()                    | ---              | ---                |
+| validateFormEvent()               | ---              | ---                |
+
+\* For Private Forms
```

Comment:
If customers are using public forms, they wont use this function. This function allows a private form to be submitted on behalf of a key. If the form is public, they can be submitted anonymously.